cli/pkg/kctrl/cmd/package/init/pkg-build.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package init
+
+import (
+	"os"
+
+	appBuild "github.com/vmware-tanzu/carvel-kapp-controller/cli/pkg/kctrl/cmd/app/init"
```

Comment:
per go conventions import directives should not be camelcase

Reply:
@sethiyash @100mik